app/src/main/res/values-tr/strings.xml

```diff
@@ -30,7 +30,7 @@
   <string name="shortcut_erase_short_label">Sil</string>
   <string name="shortcut_erase_long_label">Gezinti geçmişini sil</string>
   <string name="shortcut_erase_and_open_short_label">Sil ve aç</string>
-  <string name="shortcut_erase_and_open_long_label">Sil ve %1$s’u aç</string>
```

Comment:
Might be worth having a Turkish speaker check this. I'm not sure what the `’u` does.